autoPyTorch/pipeline/components/setup/traditional_ml/base_model.py

```diff
@@ -113,15 +130,21 @@ def predict_proba(self, X_test: Union[pd.DataFrame, np.ndarray]) -> np.ndarray:
             X_test = X_test.to_numpy()
         return self.model.predict(X_test, predict_proba=True)
 
+    def score(self, X_test: Union[pd.DataFrame, np.ndarray], y_test: Union[pd.Series, np.ndarray, List]) -> float:
+        assert self.model is not None, "Cant score without fitting first"
+        if isinstance(X_test, pd.DataFrame):
+            X_test = X_test.to_numpy()
+        return self.model.score(X_test, y_test)
```

Comment:
Can we add tests to check each input types, i.e. pd.DataFrame, np.ndarray, List, yield the same value in unittest?

Reply:
Hey, why would you like to test that? Could you tell me what could cause such an issue?